File: src/main/java/io/karma/pda/common/app/SettingsApp.java
/*
 * Copyright (C) 2024 Karma Krafts & associates
 */

package io.karma.pda.common.app;

import io.karma.pda.api.common.app.AbstractApp;
import io.karma.pda.api.common.app.AppContext;
import io.karma.pda.api.common.app.DefaultApps;

/**
 * @author Alexander Hinze
 * @since 11/04/2024
 */
public final class SettingsApp extends AbstractApp {
    public SettingsApp() {
        super(DefaultApps.SETTINGS);
    }

    @Override
    public void init(final AppContext context) {

    }
}

File: src/main/kotlin/io/karma/pda/common/app/SettingsApp.kt
/*
 * Copyright (C) 2024 Karma Krafts & associates
 */

package io.karma.pda.common.app

import io.karma.pda.api.common.app.AppType
import io.karma.pda.api.common.app.compose.ComposableApp

/**
 * @author Alexander Hinze
 * @since 13/04/2024
 */
class SettingsApp(type: AppType<*>) : ComposableApp(type) {
    override fun compose() {
        defaultView {}
    }
}